person.rb

```diff
@@ -27,4 +27,8 @@ def can_use_services?
   def correct_name
     @name
   end
+
+  def add_rental(date, person)
+    Rental.new(date, person, self)
+  end
```

Comment:
- [ ] In this `add_rental`, I think you need to pass `book` and not `person`. Also, on the line 32, the first argument will be `book`, then person (i.e. self), and last `date`.